nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/SetCommand.java

```diff
@@ -194,6 +194,320 @@ private String getResourceName() {
             }
             return dottedNameForResourceName.toString().replace('.', '/');
         }
+
+        // Methods and variables used within a call to the execute method
+        private class ExecutionContext {
+            String pattern = SetOperation.this.pattern;
+            String attrName = SetOperation.this.attrName;
+            String prefix;
+            boolean lookAtSubNodes;
+            boolean applyOverrideRules;
+
+            private Map<Dom, String> findDottedNames(TreeNode[] parentNodes, String pattern) {
+                Map<Dom, String> dottedNames = new HashMap<Dom, String>();
+                this.applyOverrideRules = false;
+                if (lookAtSubNodes) {
+                    for (TreeNode parentNode : parentNodes) {
+                        dottedNames.putAll(getAllDottedNodes(parentNode.node));
```

Comment:
https://ci.eclipse.org/glassfish/job/glassfish_build-and-test-using-jenkinsfile/view/change-requests/job/PR-24185/2/console
```
[ERROR] /home/jenkins/agent/workspace/_test-using-jenkinsfile_PR-24185/nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/admin/SetCommand.java:[211,44] non-static method getAllDottedNodes(org.jvnet.hk2.config.Dom) cannot be referenced from a static context
```

Reply:
I'm sorry, I should have checked better. I rebased against `master`, that broke my build but I forgot to rebuild again. Should be fixed by my most recent commit